tslint.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "tslint-config-standard",
   "rules": {
+    "deprecation": false,
```

Comment:
Not ideal, but there's a lot of deprecations. These can be resolved, but it's a large change (so might be better for later PRs).